doc/releases/changelog-dev.md

```diff
@@ -107,6 +107,13 @@
 * The experimental `DefaultQubit2` device now supports computing VJPs and JVPs using the adjoint method.
   [(#4374)](https://github.com/PennyLaneAI/pennylane/pull/4374)
 
+* Refactoring of `pennylane/interfaces`.  The `execute_fn` passed to the machine learning framework boundaries 
+  is now responsible for converting parameters to numpy. The gradients module can now handle tensorflow parameters,
+  but gradient tapes now retain the original dtype instead of converting to float64.  This may cause instability 
```

Comment:
Can you add the float64 part in breaking changes?

Reply:
Adding. And adding a warning to `qml.gradients.finite_diff` if float32 is detected.